x-pack/plugins/observability_solution/observability_shared/public/hooks/use_editable_settings.tsx

```diff
@@ -80,6 +80,10 @@ export function useEditableSettings(settingsKeys: string[]) {
 
   async function saveAll() {
     if (settings && !isEmpty(unsavedChanges)) {
+      let updateErrorOccurred = false;
+      const subscription = settings.client.getUpdateErrors$().subscribe((error) => {
+        updateErrorOccurred = true;
+      });
```

Comment:
Is this to capture errors from `settings.client.set`? Does `settings.client.set` not throw?

Reply:
It does not:

> Sets the value for a uiSetting. If the setting is not registered by any plugin it will be stored as a custom setting. The new value will be synchronously available via the get() method and sent to the server in the background. If the request to the server fails then a updateErrors$ will be notified and the setting will be reverted to its value before set() was called.